frontend/src/metabase/admin/datamodel/components/filters/pickers/DatePicker/RangeDatePicker.tsx

```diff
@@ -18,15 +18,13 @@ import SpecificDatePicker from "./SpecificDatePicker";
 export interface BetweenPickerProps {
   className?: string;
   filter: Filter | any[];
-  primaryColor?: string;
```

Comment:
These props are unpassed. I made this refactor well before I realized I need to theme Mantine date picker not our own date picker. But I leave the change here since this helps removes confusion.

Reply:
I double-checked the `primaryColor={...}` props in the codebase, and indeed the only usages were buttons (e.g. `DateUnitSelector`, `CurrentButton` that is not a datepicker, so we should be in the clear.